core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1016,6 +1019,12 @@ ServerBuilder serviceConfigBuilder(ServiceConfigBuilder serviceConfigBuilder) {
         return this;
     }
 
+    ServerBuilder annotatedServiceBindingBuilder(
+            AnnotatedServiceBindingBuilder annotatedServiceBindingBuilder) {
+        annotatedServiceBindingBuilders.add(annotatedServiceBindingBuilder);
```

Comment:
Shouldn't this be `virtualHostTemplate.addAnnotatedServiceBindingBuilders.add(...)`? `AnnotatedServiceBindingBuilder.applyToServiceConfigBuilder()` and `ServerBuilder.annotatedServiceBindingBuilder` could be then removed?

Reply:
> Shouldn't this be virtualHostTemplate.addAnnotatedServiceBindingBuilders.add(...)?

`ServerBuilder.annotatedServiceBindingBuilder` and `VirtualHostBuilder.addAnnotatedServiceBindingBuilders` add different classes.

> AnnotatedServiceBindingBuilder.applyToServiceConfigBuilder() and ServerBuilder.annotatedServiceBindingBuilder could be then removed?

I changed the time of creation because I had to create a `ServiceConfigBuilder` including the `extensions`. So I think this is necessary.

Do you have any other opinions?

Reply:
Ah, OK, then. :bowing_man: